e2e/nx-run/src/cache.test.ts

```diff
@@ -61,7 +62,7 @@ describe('cache', () => {
       'read the output from the cache'
     );
 
-    if (process.platform != 'linux') {
+    if (process.platform != 'linux' && !isVerboseE2ERun()) {
```

Comment:
Verbose logging is changing the terminal output leading to failed test

Reply:
So `expectMatchedOutput` doesn't work with the verbose logging?

Shouldn't we fix this test so that it works even with verbose logging?

Reply:
Not sure what you mean by doesn't work. The output works, but because of the `--verbose` we are not printing out compact view (containing expected `read the output from the cache`) but full output which has `local cache` next to tasks that were retrieved from the cache.

Code works as expected, but test does not take into account different environment in which we run it.